CODEOWNERS

```
@@ -1 +1 @@
-*       @elatt @oleksandr-saienko @dulcardo @alexromanyuk
+*       @elatt @oleksandr-saienko @dulcardo @alexromanyuk @andrius-senulis
```

```suggestion
*       @elatt @alexromanyuk @andrius-senulis
```
Dulcardo and Alex are no longer with the company.

---

Thanks, updated